specfile/specfile.py

```diff
@@ -104,3 +113,66 @@ def tags(self, section: str = "package") -> Iterator[Tags]:
                 yield tags
             finally:
                 raw_section.data = tags.reassemble()
+
+    @contextlib.contextmanager
+    def changelog(self) -> Iterator[Optional[Changelog]]:
+        """
+        Context manager for accessing changelog.
+
+        Yields:
+            Spec file changelog as `Changelog` object or None if there is no %changelog section.
+        """
+        with self.sections() as sections:
```

Comment:
Hm, this means that the changelog has to be parsed from scratch every time that the user of the library wants to change it, right? I had an idea for an alternative approach though I am not sure if it's really better. We could make `Changelog` inehrit from `Section` and it could have an `entries` attribute that would contain the list of entries similarly to the way it is now. This way, I guess it would be a bit easier to keep the two "views" on the changelog section (raw section and a list of changelog entries) in sync without having to parse it each time. What do you think about this approach?

Reply:
Makes sense, lets keep it the way as with `Tags`, thanks for the explanation